sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -1092,6 +1092,58 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(ElementAt(mb0, Literal(Array[Byte](3, 4))), null)
   }
 
+  test("SPARK-26965 correctly handles ElementAt nullability for arrays") {
```

Comment:
Since this is a new feature, it seems that we don't need JIRA ID, `SPARK-26965`, in the test case name.

Reply:
I dropped that in the latest commit though, any rule for that?

Reply:
Yes. I've heard the rule from @cloud-fan in my old PRs.

I'm not sure if that is a written rule or not. 

Hi, @cloud-fan . Do we have some URLs for the above `SPARK JIRA ID` usage rule in the test case name?

Reply:
oh, I see. I'll be more careful next time, thanks!